seungjong/app.js

```diff
@@ -47,11 +47,74 @@ app.post('/signUp', async (req, res, next) => {
             profile_image
         ) VALUES (?, ?, ?, ?);
         `,
-        [name, email, password, profileImage]
+        [name, email, password, prfileImage]
     );
-    res.status(201).json({ message: "sucessfully created!" });
+    res.status(200).json({ message: "sucessfully created!" });
 })
 
+app.post('/addPost', async (req, res, next) => {
```

Comment:
`/addPost`라고 작성해주신 엔드포인트의 api는 post를 등록하는 기능의 api인 것 같습니다! 다만 엔드포인트를 작성하실 때에는 add라는 액션(동작)이 들어가기보다는 정확히 목적하는 리소스만을 작성하고 해당 api의 동작은 http request method로서 표현되는 것이 REST한 api를 설계하는 방법 입니다. 자세한 내용은 차주 세션에서 다룰 예정이니 지금은 엔드포인트에 동작을 의미하는 동사가 들어가지 않는다만 기억해주세요!
```suggestion
app.post('/post', async (req, res, next) => {
```